styles/layout.css

```diff
@@ -0,0 +1,66 @@
+
+
+
+#menu {
+  margin: 0;
+  float: left;
```

Comment:
I think this float will get ignored because you're using `display: inline-block;`